.circleci/config.yml

```diff
@@ -16,15 +16,20 @@ jobs:
 
       - run: npm install
       - run: npm run linter:check
-      - run: npm run release:test
-      - run: node build.js '0.0.1'
+      - run: npm run build:package
+      - run: npm run build:storybook
+      # check npm token exists
+      - run: echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > ~/repo/.npmrc
+      - run: cat ~/repo/.npmrc
```

Comment:
better to not leak token in this way :) can it be even not saved at all?

Reply:
Hmm I can remove it no problem but is it really a leak? The output afaik only get's outputted into circle ci which requires auth to view. I mean really someone could ssh into the container and just cat out the credentials. However, we do need to save it so we can publish.

Reply:
logs usually available and viewable for oss projects. But even with auth - we should not show token to everyone with gh access from the early days. Otherwise, we should reset it every time someone got fired :)